app/models/organisation.rb

```diff
@@ -165,6 +165,10 @@ def generate_searchable_content
     ).tsvector
   end
 
+  def should_generate_new_friendly_id?
```

Comment:
Non-block suggestion to add a bit of context on this call:
```suggestion
  # Friendly_id method to control whether new friendly IDs are created when a model is updated
  def should_generate_new_friendly_id?
```